ElrondNetwork/api.elrond.com
# Improve node sort predictability

## Reasoning
- Node sorting criteria works differently between runs depending whether it was stored in local cache or not
  
## Proposed Changes
- Instead of using `in` operator to check whether the sort attribute exists in node, check explicity for undefined / string empty

## How to test (mainnet)
- `/nodes?online=true&size=6000&sort=locked&order=desc` should always return ~3200. Before it was returning on the first call ~5500, then for every subsequent call, ~3200
